Ukoly/DU_7/Rodne_cislo.py

```diff
@@ -0,0 +1,101 @@
+MONTH_30 = [4, 6, 9, 11]
+MONTH_31 = [1, 3, 5, 7, 8, 10, 12]
+
+
+def main():
+    number = input('Enter the birth number: ')
+    while len(number) != 11:
+        print('The number has to be 11 characters long')
+        number = input('Enter the birth number: ')
+
+    return number
+
+
+def format(birth_number):
+    print(birth_number)
+    separate = birth_number.split('/')
+    without_slash = ''.join(separate)
+    try:
+        int(without_slash)
+    except ValueError:
+        print('You can use only digits or slash "/"')
+    try:
+        birth_number.count('/') == 1
+        if birth_number.count('/') > 1:
+            raise ValueError('There should be only one separator')
+    except ValueError:
+        print('There should be only one separator')
+        return False
+    try:
+        birth_number[6] == '/'
+        if birth_number[6] != '/':
+            raise ValueError('Separator has to be slash "/" in 7th position')
+    except ValueError:
+        print('Separator has to be slash "/" in 7th position')
+        return False
+    return True
+
+
+def divisibility(birth_number):
+    separate = birth_number.split('/')
+    without_slash = ''.join(separate)
+    try:
+        int(without_slash) % 11 == 0
+        if int(without_slash) % 11 != 0:
+            raise ValueError('The number hast to be divisible by 11')
+    except ValueError:
+        print('The number hast to be divisible by 11')
+        return False
+    else:
```

Comment:
Aaaa ja moc nemusím `else` vetvu na `try` bloku... ale aj keď si to odmyslím... je to zbytočné, stačilo by mať na konci funkcie tento `return` a bude to robiť to isté a ako bonus tam nebude `else` :smile: